libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -44,7 +45,7 @@ export class ReadingListEffects implements OnInitEffects {
     this.actions$.pipe(
       ofType(ReadingListActions.removeFromReadingList),
       concatMap(({ item }) =>
-        this.http.delete(`/api/reading-list/${item.bookId}`).pipe(
+        this.http.delete(`${AppConstants.readingApi}/${item.bookId}`).pipe(
```

Comment:
Implemented service for deleting a book from reading list

Reply:
what is the name of service which you implemented